test/test_user.py

```
@@ -0,0 +1,26 @@
+
+    payload = {"first_name": "Nome Atualizado"}
+    response = await client.patch("/api/v1/users/me", json=payload)
+
+    assert response.status_code == 200
+    assert response.json()["first_name"] == "Nome Atualizado"
+
+
+@pytest.mark.asyncio
+async def test_user_updated_telegram_id(cliente_autenticado):
+
+    client, usuario = cliente_autenticado
```

Ruff (0.15.0)

[warning] Unpacked variable `usuario` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)